src/stirling/source_connectors/socket_tracer/bcc_bpf_intf/grpc_c.h

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2018- The Pixie Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+// The amount of bytes in a single slice of data.
+// This value was not chosen according to some constant in the grpc-c library.
+// Largest I saw was 1293.
+#define GRPC_C_SLICE_SIZE (16380)
+
+// This needs to not be lower than 8 (which is the maximum amount of inlined
+// slices in a grpc_slice_buffer). The real maximum size isn't known - it can
+// probably be larger than 8. Until now I have not seen a size larger than 2 used,
+// so 8 is more than enough.
+#define SIZE_OF_DATA_SLICE_ARRAY (8)
+
+#define GRPC_C_DEFAULT_MAP_SIZE (10240)
+
+#define MAXIMUM_AMOUNT_OF_ITEMS_IN_METADATA (30)
+#define MAXIMUM_LENGTH_OF_KEY_IN_METADATA (44)
+#define MAXIMUM_LENGTH_OF_VALUE_IN_METADATA (100)
+
+enum grpc_c_version_t {
+  GRPC_C_VERSION_UNSUPPORTED = 0,
+  GRPC_C_V1_19_0,
+  GRPC_C_V1_24_1,
+  GRPC_C_V1_33_2,
+  GRPC_C_V1_41_1,
+  GRPC_C_VERSION_LAST
+};
+
+struct grpc_c_data_slice_t {
+  uint32_t slice_len;
```

Comment:
nit: Just call it 'len' since it's already inside grpc_c_data_slice_t with a slice in the type name.

Reply:
Done